src/SFA.DAS.ApprenticeCommitments.Api/AppStart/AddServiceRegistrationExtension.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.Api.Common.Infrastructure;
+using SFA.DAS.Api.Common.Interfaces;
+using SFA.DAS.ApprenticeCommitments.Application.Services;
+using SFA.DAS.SharedOuterApi.Infrastructure;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.ApprenticeCommitments.Api.AppStart
+{
+    public static class AddServiceRegistrationExtension
+    {
+        public static void AddServiceRegistration(this IServiceCollection services)
+        {
+            services.AddHttpClient();
+            services.AddTransient<IAzureClientCredentialHelper, AzureClientCredentialHelper>();
+
+            services.AddTransient(typeof(IInternalApiClient<>), typeof(InternalApiClient<>));
+            services.AddTransient<ApprenticeCommitmentsService>();
```

Comment:
Do we need to wire this up with a supporting Interface

Reply:
See https://github.com/SkillsFundingAgency/das-apim-endpoints/pull/232#discussion_r556401338